### ISSUE TITLE
CNTK_202_Language_Understanding loss & metric 

### ISSUE THREAD

**Issue by mismalinowy** [Microsoft/CNTK#1192]
Hi,

I set up CNTK on Windows. I am trying to run the language understanding sample. https://github.com/Microsoft/CNTK/blob/master/Tutorials/CNTK_202_Language_Understanding.ipynb and i have problem.

cntk returns:
Finished Epoch [16]: [Training] loss = 2837788428.932164 * 18014, metric = 98.4% * 18014 6.094s (2956.1 samples per second).
when i use : https://github.com/Microsoft/CNTK/wiki/Hands-On-Labs-Language-Understanding 
everything is ok, but on python is problem.

I use:
python 3.4 anaconda
windows 10 
vs 13 update 5

**Comment by mahilleb-msft** [comment]
This is already some time ago; with current build and the current version of the notebook everything looks good. I'm closing the issue for now, but if you find the issue persists, please re-open (or file a new one)!
Thanks, Mark
